gnd/src/main/java/com/google/android/gnd/persistence/local/LocalDataStore.java

```diff
@@ -114,4 +114,7 @@
 
   /** Returns the tile with the specified id from the local data store, if found. */
   Maybe<Tile> getTile(String tileId);
+
+  /** Deletes the tile with the specified id from the local data store, if found.*/
+  Maybe<Integer> deleteTile(String tileId);
```

Comment:
Remove this. I originally deleted tiles from the DB upon file removal, but it's actually much simpler to just retain them and update their state.

Reply:
Did you mean to add this as a TODO?